backend/tessend/tessend.go

```diff
@@ -20,72 +20,80 @@ import (
 
 const tessenURL = "https://tessen.sensu.io/v2/data"
 
-// Tessend is the tessen daemon
+// Tessend is the tessen daemon.
 type Tessend struct {
 	interval  uint32
 	store     store.Store
 	ctx       context.Context
 	cancel    context.CancelFunc
 	errChan   chan error
```

Comment:
We have a few daemons that don't make use of the `errChan`, however the function `Err() <-chan error` is required to satisfy the `Daemon` interface.

Reply:
That means you can just return a closed channel from the method instead of keeping the struct field.

Reply:
I'm not sure backend.go expects that the channel will be closed. Because the backend appends all daemons to the error group, it errors on the closed channel.

Reply:
The selector in backend.go will just wait forever on a closed channel, which is the same behaviour as never writing a value to it.

Reply:
Oh, I see, there is a bug in the errGroup construct.